src/features/layers/layersSlice.spec.js

```diff
@@ -41,7 +41,6 @@ describe('layers reducer', () => {
     backtrackPct: 0,
     drawPortionPct: 100,
     effect: false,
-    rotateCompleteLoop: false,
```

Comment:
We needed this to fix some bug, right? Is this just a hack that we don't need anymore? The default for all shapes is to have it set to false, but then several shapes turn it on.

Reply:
I don't think we need it any more. I tested each affected shape, but there's a chance I missed something obviously.